test/acceptance/keywords/content_assertions.robot

```diff
@@ -119,15 +119,20 @@ Page Should Not Contain Element With Disabling Source Logging
 
 Element Should Contain
     Element Should Contain    some_id    This text is inside an identified element
+    Element Should Contain    some_id    THIS TEXT IS INSIDE AN IDENTIFIED ELEMENT  ignore_case=True
     Run Keyword And Expect Error
     ...    Element 'some_id' should have contained text 'non existing text' but its text was 'This text is inside an identified element'.
     ...    Element Should Contain    some_id    non existing text
     Run Keyword And Expect Error
     ...    Element with locator 'missing_id' not found.
     ...    Element Should Contain    missing_id    This should report missing element.
+    Run Keyword And Expect Error
+    ...    Element 'some_id' should have contained text 'foobar' but its text was 'This text is inside an identified element'.
+    ...    Element Should Contain    some_id    foobar    ignore_case=True
 
 Element Should Not Contain
     Element Should Not Contain    some_id    This text is not inside an identified element
+    Element Should Not Contain    some_id    THIS TEXT is not inside an identified element   ignore_case=False
```

Comment:
1) Please add line: `Element Should Not Contain | some_id | This text is not inside an identified element | ignore_case=True |`
2) Please also add new test where `Element Should Not Contain` when `ignore_case`argument is used with True and False values but the keyword fails. In the test use `Run Keyword And Expect Error`to verify that correct error is raised.